test/test.do

```diff
@@ -191,6 +191,17 @@ graph drop g3
 
 graph drop _all
 
+*Interaction Weighted Estimator (Sun and Abraham, 2021) 

```

Comment:
This is clearer, use this title for readme.md @Constantino-Carreto-Romero

Reply:
@jorpppp, I changed it for "Sun and Abraham Estimator" in https://github.com/JMSLab/xtevent/commit/602cf1d6bbdbb4083f9cba70c6b0d47b3f158c51, but I can still change it for the whole reference "Interaction Weighted Estimator (Sun and Abraham, 2021)".